gen/admi_002_001_01/validate.go

```diff
@@ -1,9 +1,9 @@
 package admi_002_001_01
 
-func (d Document) Validate() error {
+func (d DocumentTCH) Validate() error {
 	return d.Admi00200101.Validate()
```

Comment:
Are we really using these validations? most are stubs, but it might be possible to auto-generate them from the schema.

Reply:
The idea was to implement/generate Validate() functions.

Reply:
I am working on auto-generating them from the XSD